### DISCUSSION TITLE
The package @vitest/snapshot no release.

### DISCUSSION THREAD

**Discussion by veaba** [vitest-dev/vitest#3153] [Q&A] [answered]
I was testing `packages/browser` and found that `@vitest/snapshot` is not published to npm.

**Comment by sheremet-va** [comment]
Yes, it is not published yet.

**Reply by veaba** [reply]
Hi
Is this `@vitest/snapshot` scheduled for release recently?

 I would like to know when it will be available, thank you for your support.

**Reply by sheremet-va** [reply, marked as answer]
Should be released until the end of the week
